tests/nlu/training_data/formats/test_rasa_yaml.py

```diff
@@ -10,7 +10,6 @@
 from rasa.nlu.training_data.formats.rasa_yaml import RasaYAMLReader, RasaYAMLWriter
 
 MULTILINE_INTENT_EXAMPLES = f"""
```

Comment:
This is just a normal string instead of f-string now (same for all the other examples below).